src/m365/spo/commands/web/web-set.spec.ts

```diff
@@ -444,6 +454,34 @@ describe(commands.WEB_SET, () => {
     await command.action(logger, { options: { url: 'https://contoso.sharepoint.com/sites/team-a', footerEnabled: false } });
   });
 
+  it('enables navAudienceTargetingEnabled', async () => {
+    sinon.stub(request, 'patch').callsFake((opts) => {
+      if (JSON.stringify(opts.data) === JSON.stringify({
```

Comment:
Instead of doing it this way, I suggest that we perform a body check in the assert, this will result in a much cleaner error when the test fails.

Example:

https://github.com/pnp/cli-microsoft365/blob/97d58e0ab2ebff35aad1870901e18702b7ff85aa/src/m365/aad/commands/user/user-guest-add.spec.ts#L138-L150